precedence.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+    "use strict";
+
+    function Precedence() {
+        this.numbers = [];
+        this.operations = [];
+        this.turnOn = false;
+        this.finalResult = 0;
+    }
+
+    Precedence.prototype.returnValue = function() {
+        this.result = Number(this.numbers[0]);
```

Comment:
Why are you using field, when a local variable would do the job? You are 'polluting' the instance.

Reply:
Do not store anything but number in *numbers* array. If you need to parse something, parse it when you are loading from DOM.